product_docs/docs/biganimal/release/getting_started/index.mdx

```diff
@@ -12,12 +12,13 @@ navigation:
 ---
 
 As a cloud administrator, you can:
--  Set up BigAnimal with your existing Azure subscription or AWS account.
+-  Set up BigAnimal with your existing Azure subscription or AWS account. (This isn't necessary if you are using BigAnimal's cloud account.)
 -  Invite others to join you in exploring what EDB has to offer.
 -  Create initial clusters as an account owner so that development can begin. 
 
 <!-- Alternatively, you can use [BigAnimal's Terraform provider](../using_cluster/terraform_provider) to incorporate BigAnimal into your existing cloud infrastructure workflows.  -->
 
 If you purchased BigAnimal directly from EDB Sales, you need to [set up your own identity provider](/biganimal/release/getting_started/identity_provider). If you purchased from Azure Marketplace, you need to [set up your Azure account](/biganimal/release/getting_started/02_azure_market_setup). 
 
-After setting up your organization through your identity provider or Azure Marketplace account, you connect your cloud account to BigAnimal.
\ No newline at end of file
+After setting up your organization through your identity provider or Azure Marketplace account, you connect your cloud account to BigAnimal.
```

Comment:
"After setting up your organization through your identity provider or Azure Marketplace account," this is not the assumption to "connect your cloud account to BigAnimal" anymore.

So this line should be removed.

Today, as long as you can login and you are org-owner, you can connect your account to BigAnimal anytime.